recipes/libsemigroups/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "libsemigroups" %}
+{% set version = "0.2.1" %}
+{% set md5 = "de073733a7d389e6ce378f4586ff1f42" %}
+{% set repo = "https://github.com/james-d-mitchell/libsemigroups" %}
+{% set home = "https://james-d-mitchell.github.io/libsemigroups/" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: {{ repo }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  #git_rev: v{{ version }}
+  #git_url: {{ repo }}.git
+  #git_tag: autotools
+
+build:
+  number: 0
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/libsemigroups  # [unix]
+    - test -f ${PREFIX}/include/libsemigroups/semigroups.h  # [unix]
```

Comment:
Ok. I wasn't sure how to do this in a portable way. Do you have a recommendation or example recipe?